core/integration/module_test.go

```diff
@@ -5873,146 +6076,3 @@ func logGen(ctx context.Context, t *testctx.T, modSrc *dagger.Directory) {
 		}
 	})
 }
-
-func (ModuleSuite) TestSSHAgentConnection(ctx context.Context, t *testctx.T) {
```

Comment:
it's been driving me nuts that these are below the test helpers so i moved em up